### DISCUSSION TITLE
Public access to zwave statistics

### DISCUSSION THREAD

**Discussion by ivdimitro** [zwave-js/zwave-js#3869] [Q&A] [answered]
If I am not mistaken when the reporting feature was introduced, there was a website where the statistics were displayed? But I can not find the site in the documentation. 
Is it this site still working or am I misremembering something? 
Thanks.

**Comment by AlCalzone** [comment, marked as answer]
> there was a website where the statistics were displayed

Not publicly, no. We share selected statistics from time to time in our Slack, but opening the (pretty heavy) dashboard to the public would require a more powerful server and isn't feasible currently.

**Reply by ivdimitro** [reply]
Thanks. 
BTW some device manufacturers will likely be interested in this data. Have you considered to license it in order to cover the server cost and make it publicly available?

**Reply by AlCalzone** [reply]
We're planning to use some of it to convince manufacturers to work closer with us and improve the Z-Wave ecosystem. @blhoward2 has some plans here.
Licensing sounds terribly like "selling user data" though, which isn't going to sit well, especially with HomeAssistant users.

**Reply by blhoward2** [reply]
Licensing it also opens up a can of worms under various privacy laws. Not really worth it.
